plugins/woocommerce-admin/client/products/fields/attribute-field/add-attribute-modal.tsx

```diff
@@ -171,38 +165,49 @@ export const AddAttributeModal: React.FC< CreateCategoryModalProps > = ( {
 																'Search or create attribute',
 																'woocommerce'
 															) }
-															value={ attribute }
+															value={
+																formAttr.id &&
+																formAttr.name
+																	? {
+																			id: formAttr.id,
+																			name: formAttr.name,
+																	  }
+																	: null
```

Comment:
Why not just pass in `formAttr` if it contains `id` and `name` already?

Reply:
I actually went this direction since I was getting a TS error:
![image](https://user-images.githubusercontent.com/444632/199120813-921c6f2a-5767-4601-b7a1-219a5eec990c.png)

I'm not sure _why_ this error is appearing, since I should be ensuring that `id` is defined with the ternary check, but the error went away when I narrowed the object. Do you know offhand why that might be happening? I wasn't sure I wanted to spend more time troubleshooting it when this solution worked.